scripts/process_data.py

```diff
@@ -630,6 +642,68 @@ def get_matching_summary(num_intial_frames: int, num_matched_frames: int) -> str
     return f"[bold green]COLMAP found poses for {num_matched_frames / num_intial_frames * 100:.2f}% of the images."
 
 
+def polycam_to_json(
+    image_filenames: List[Path],
+    cameras_dir: Path,
+    output_dir: Path,
+    min_blur_score: float = 0.0,
+    crop_border_pixels: int = 0,
+) -> List[str]:
+    """Convert Polycam data into a nerfstudio dataset.
+
+    Args:
+        image_filenames: List of paths to the original images.
+        cameras_dir: Path to the polycam cameras directory.
+        output_dir: Path to the output directory.
+        min_blur_score: Minimum blur score to use an image. Images below this value will be skipped.
+        crop_border_pixels: Number of pixels to crop from the border of the image.
+
+    Returns:
+        Summary of the conversion.
+    """
+
+    frame_json = io.load_from_json(cameras_dir / f"{image_filenames[0].stem}.json")
+
+    data = {}
+    data["fl_x"] = frame_json["fx"]
+    data["fl_y"] = frame_json["fy"]
+    data["cx"] = frame_json["cx"] - crop_border_pixels / 2
+    data["cy"] = frame_json["cy"] - crop_border_pixels / 2
+    data["w"] = frame_json["width"] - crop_border_pixels
+    data["h"] = frame_json["height"] - crop_border_pixels
+    data["camera_model"] = CAMERA_MODELS["perspective"].value
+    data["orientation_override"] = "none"
```

Comment:
Might be good to make a note that this needs to be the string literal "none", as `camera_utils.auto_orient_and_center_poses` checks against the string.